src/Thunks/api-ms-win-core-processthreads.hpp

```diff
@@ -158,6 +158,44 @@ namespace YY
 #endif
 
 
+#if (YY_Thunks_Support_Version < NTDDI_WS03)
+
+		//Windows Vista, Windows XP Professional x64 Edition [desktop apps only]
+		//Windows Server 2008, Windows Server 2003 with SP1 [desktop apps only]
+		__DEFINE_THUNK(
+		kernel32,
+		4,
+		BOOL,
+		WINAPI,
+		SetThreadStackGuarantee,
+			_Inout_ ULONG *StackSizeInBytes
+		)
+		{
+			if (auto pSetThreadStackGuarantee = try_get_SetThreadStackGuarantee())
+			{
+				return pSetThreadStackGuarantee(StackSizeInBytes);
+			}
+			else
+			{
+				ULONG prev_size = ((TEB*)NtCurrentTeb())->GuaranteedStackBytes;
```

Comment:
Windows XP SP3  `TEB::GuaranteedStackBytes` (0xf78) not found, 
I only found this member `0xf78 Spare3           : Uint4B`

But you use it here, is there no problem with doing so???